ruby/vampires.rb

```diff
@@ -56,5 +56,4 @@
   end

```

Comment:
Line 47:

For your while loop lets add a condition that will help the loop break naturally.  The functionality is definitely working in this loop but `break` is a little abrupt.  We always want a flow to our loops so providing a variable to evaluate would be a good refactor idea here.